website/package.json

```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.2.0",
-    "@docusaurus/preset-classic": "2.2.0",
+    "@docusaurus/core": "^2.4.3",
+    "@docusaurus/preset-classic": "^2.4.3",
```

Comment:
The recommended way to include Docusaurus packages in the project is to pin exact versions rather than using a range qualifier (`^` here). This prevents the two packages from going out of sync with each other.

You can read the Docusaurus upgrade guide here: https://docusaurus.io/docs/installation#updating-your-docusaurus-version